sys/include/net/gcoap.h

```diff
@@ -403,6 +423,33 @@ extern "C" {
 #ifndef CONFIG_GCOAP_PORT
 #define CONFIG_GCOAP_PORT              (5683)
 #endif
+#ifndef CONFIG_GCOAPS_PORT
```

Comment:
```suggestion
/**
 * @brief   Secure Server port; use RFC 7252 default if not defined
 */
#ifndef CONFIG_GCOAPS_PORT
```

Reply:
Done. Also documented the `coap_socket_t` struct now.